AuxTel/AuxTel-Non-standard-Operations/AuxTel-Emergency-Shutdown/AuxTel-Emergency-Shutdown.rst

```diff
@@ -0,0 +1,103 @@
+- Eventually, the dome shutter is closed and all equipment is safe.
+
+.. _Title-of-Procedure-Procedure-Steps:
+Procedure Steps
+===============
+
+.. todo::
+   Please use the standard safety measures when visiting the dome enclosure. **Safety of personnel always goes first.**
+
+.. This section should include the procedure. There is no strict formatting or structure required for procedures. It is left to the authors to decide which format and structure is most relevant.
+.. In the case of more complicated procedures, more sophisticated methodologies may be appropriate, such as multiple section headings or a list of linked procedures to be performed in the specified order.
+.. For highly complicated procedures, consider breaking them into separate procedure. Some options are a high-level procedure with links, separating into smaller procedures or utilizing the reST ``include`` directive <https://docutils.sourceforge.io/docs/ref/rst/directives.html#include>.
+
+In the event of an emergency dome closure, the dome shutter won't be able to be operated from the standard CSC controls. The AuxTel dome would need to be closed manually. In case of sudden inclement weather, be very cautious and start closing early.
+
+.. warning::
+    If you are dealing with a dome shutter emergency shutdown. Proceed calmly  to keep the equipment safe.
+
+.. _Title-of-Procedure-Critical-Step-1:
+
+Shutter Closure
+---------------
+
+1. Go up to the AuxTel dome (second floor).  Locate the control box at the top of the stairs.  The top black button should close the shutter.
+
+.. figure:: /AuxTel/AuxTel-Non-standard-Operations/_static/Box_at_Top_of_Stairs.jpg
+  :name: AuxTel control box
+
+2. If the above step fails, locate the AuxTel cRIO box. This box rotates with the AuxTel dome. Open it and identify the cRIO switches.
+
+.. figure:: /AuxTel/AuxTel-Non-standard-Operations/_static/cRIO_box.jpg
+  :name: AuxTel cRIO
+
+.. figure:: /AuxTel/AuxTel-Non-standard-Operations/_static/Switches_in_cRIO_box.jpg
+  :name: Switches inside AuxTel cRIO
+
+With **EXTREME CAUTION** slide the switch labeled "CLOSE" (yellow arrow) progresivily to the left. This is a low-level control switch and there are no limit switches or fail-safes. It is best to stop the shutter movement about 1 cm before it is fully closed to prevent you over drive the shutter until it bangs into the lower shutter or cause damage on the drive.
+
+.. _Title-of-Procedure-Critical-Step-2:
+
+Shutter Closure without Power
+-----------------------------
+
+1. In the absence of electrical power, the shutter must be closed manually. On AuxTel first floor there is a crank that fits into a ring at the top of the dome. Rotating the ring manually with the crank assistance close the shutter. Note this is a long and physically demaning procedure and the hand crank just does not have enough torque to lift the weight of the shutter if it is all the way open.
+
```

Comment:
Add this line: If you cannot safely insert the crank into the drive motor and operate the crank, then remove the crank and do not proceed.

Reply:
Added.